src/main/resources/assets/computercraft/lua/rom/apis/parallel.lua

```diff
@@ -55,12 +87,21 @@ local function runUntilLimit( _routines, _limit )
     end
 end
 
+function createBundle( ... )
+    local bundle = {}
+    bundle.data = {
+        run = true
+    }
+    bundle = setmetatable(bundle, external)
```

Comment:
`local bundle = setmetatable({ data = { run = true } }, external)`

Reply:
Or if you think this will be extended in the future, at least 
```lua
local bundle = {
  data = { run = true }
}
```

Reply:
Amended this locally, thank you!